astroquery/esa/jwst/core.py

```diff
@@ -39,6 +39,12 @@
 __all__ = ['Jwst', 'JwstClass']
 
 
+# We do trust the ESA tar files, this is to avoid the new to Python 3.12 deprecation warning
+# https://docs.python.org/3.12/library/tarfile.html#tarfile-extraction-filter
+if hasattr(tarfile, "fully_trusted_filter"):
```

Comment:
What exactly was the warning that you saw before this patch? I cannot find it in the logs. (Though it does looks like you finally have to drop astropy-helpers.)

Also, I am not sure about this. Can we *always* trust any TAR files that are given?

Reply:
It's a new python 3.12 deprecation:

```
DeprecationWarning: Python 3.14 will, by default, filter extracted tar archives and reject files or modify their metadata. Use the filter argument to control this behavior.
```

Reply:
Hmm, would the archive people be able to provide a way to inspect and decide if the TAR is trusted or not?

Anyway, I think this is okay for now but should probably have a follow-up issue to revisit.

Reply:
(I'll deal with the helpers after the 0.4.7 release, and when switching to a date based versioning scheme)